app/Http/Controllers/CaseStudiesController.php

```diff
@@ -24,7 +24,10 @@ public function get(Factory $factory, string $slug): View
 
         $view = preg_replace('/\.blade\.php$/', "", $caseStudy->template);
```

Comment:
Wouldn't `Str::replaceLast('.blade.php', '', $caseStudy->template);` more verbose?

Reply:
Or maybe it would be better to store template name only in database?